src/domain/deliveryLog.js

```diff
@@ -1,5 +1,24 @@
 import dbClient from '../utils/dbClient.js'
 
+export async function getDeliveryLogs() {
+  const DeliveryLogs = await dbClient.DeliveryLog.findMany({
+    include: { user: { include: { profile: true } }, lines: true }
```

Comment:
can you expand this into multiple lines for readability please

Reply:
Updated